src/coaster.cpp

```diff
@@ -721,6 +721,22 @@ int CoasterInstance::GetFirstPlacedTrackPiece() const
 	return -1;
 }
 
+std::vector<XYZPoint16> CoasterInstance::get_all_positions() {
```

Comment:
This clearly lacks documentation. Should not be here but in the base class though, and since it's a pure virtual, there should be a documentation block separate from the body.
It's too long ago, but I am guessing that `RemoveAllPeople` has one too, as that function is also pure virtual, it seems.

The name of the function is in the wrong style, it should be `GetAllPositions`.

Not sure if 'positions' is the right word, `PiecePositions` comes to mind as an alternative, or `Voxels`.
Also, you should check if you got everything, afaik a piece could be bigger than one voxel.